workflow.rst

```diff
@@ -366,6 +366,15 @@ name.
     * ``workflow.workflow``: all workflows;
     * ``workflow.state_machine``: all state machines.
 
+    Note that workflow configurations are attached to tags under the
```

Comment:
not the whole configuration, only the metadata.
For example, with

* https://github.com/lyrixx/SFLive-Paris2016-Workflow/blob/master/config/packages/workflow.yaml
* and `dd($container->findTaggedServiceIds('workflow'));`

you got:

```
^ array:7 [
  "workflow.article" => array:1 [
    0 => array:2 [
      "name" => "article"
      "metadata" => array:1 [
        "title" => "Manage article"
      ]
    ]
  ]
  "state_machine.task" => array:1 [
    0 => array:2 [
      "name" => "task"
      "metadata" => []
    ]
  ]
  "workflow.straight" => array:1 [
    0 => array:2 [
      "name" => "straight"
      "metadata" => []
    ]
  ]
  "workflow.round_trip" => array:1 [
    0 => array:2 [
      "name" => "round_trip"
      "metadata" => []
    ]
  ]
  "workflow.or" => array:1 [
    0 => array:2 [
      "name" => "or"
      "metadata" => []
    ]
  ]
  "workflow.and" => array:1 [
    0 => array:2 [
      "name" => "and"
      "metadata" => []
    ]
  ]
  "workflow.wtf" => array:1 [
    0 => array:2 [
      "name" => "wtf"
      "metadata" => []
    ]
  ]
]
```

Reply:
Reworded to point to the workflow metadata section also. Thanks 🙂